pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotHelixTaskResourceManager.java

```diff
@@ -427,29 +498,161 @@ private static String getTaskType(String name) {
     return name.split(TASK_NAME_SEPARATOR)[1];
   }
 
-  public static class TaskCount {
-    private int _waiting;   // Number of tasks waiting to be scheduled on minions
-    private int _error;     // Number of tasks in error
-    private int _running;   // Number of tasks currently running in minions
-    private int _total;     // Total number of tasks in the batch
+  @JsonPropertyOrder({"taskState", "subTaskCount", "startTime", "executionStartTime", "subTaskInfos"})
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  public static class TaskDebugInfo {
+    private String _startTime;
+    private String _executionStartTime;
+    private TaskState _taskState;
+    private TaskCount _subTaskCount;
```

Comment:
(nit) I think `subtask` is a word, so name the variable `_subtaskCount` and `_subtaskInfos`? Same for the methods

Reply:
changed subTask/SubTask to subtask/Subtask